README.md

```diff
@@ -45,14 +45,14 @@ Follow these steps to setup and run this code pattern. The steps are described i
 
 1. [Sign up for Watson Studio](#1-sign-up-for-watson-studio)
 1. [Create a project and add services](#2-create-a-project-and-add-services)
-1. [Create a notebook](#3-create-the-notebook)
-1. [Load customer data in the notebook](#4-load-customer-data)
-1. [Add a Watson Machine Learning service](#5-add-watson-machine-learning)
-1. [Build and run the recommendation PixieApp](#6-build-the-pixieapp)
+1. [Create a notebook](#3-create-a-notebook)
+1. [Load customer data in the notebook](#4-load-customer-data-in-the-notebook)
+1. [Add the Watson Machine Learning credentials to the notebook](#5-add-watson-machine-learning-credentials-to-the-notebook)
```

Comment:
It appears the HREF is broken. It should be `#5-add-the-watson-machine-learning-credentials-to-the-notebook`